signer/migrations/0003__create_tables.sql

```diff
@@ -196,6 +196,8 @@ CREATE TABLE sbtc_signer.sweep_transactions (
     amount BIGINT NOT NULL,
     -- The fee paid for the transaction.
     fee BIGINT NOT NULL,
+    -- The _vsize_ of the transaction in bytes.
```

Comment:
I would rephrase this to say something like 'The bitcoin "virtual size" of the transaction'. I don't think that saying that it is in bytes is incorrect.

Reply:
Yeah you're right, since witness data is 1/4 weight

Reply:
[77d4884](https://github.com/stacks-network/sbtc/pull/810/commits/77d4884094bc91a71e5c7ee44a96cc22b72b895e)